portfolio/src/main/webapp/script.js

```diff
@@ -34,8 +34,8 @@ function addRandomFact() {
 /**
  * Callback when window is scrolled
  */
-window.onscroll = OnWindowScrolled;
-function OnWindowScrolled() {
+window.onscroll = onWindowScrolled;
+function onWindowScrolled() {
     const headerHeight = document.getElementById("header").offsetHeight;
     const scrolledY = window.pageYOffset;
     const navBar = document.getElementById("navbar");
```

Comment:
It is probably better to use same quotation marks in the document. In google codestyle it is single quotes ([link](https://google.github.io/styleguide/jsguide.html#features-string-literals))

Reply:
You're right, thank you! I edited that.